app/views/static/index.html.erb

```diff
@@ -54,7 +54,7 @@
       <section class="caption">
         <div class="row">
           <h1 class="superbar" style="text-align: center;">
-          <img id="MastheadLogo" src="img/diaper-base-logo-full-white-text.svg" style="width: 70%; height: 70%; " alt="Diaperbase" title="Diaperbase" class="serv_icon"/>
+          <img id="MastheadLogo" src="/img/diaper-base-logo-full-white-text.svg" style="width: 70%; height: 70%; " alt="Diaperbase" title="Diaperbase" class="serv_icon"/>
```

Comment:
This wasn't causing any issues since it was the home page, but if it was on a route (e.g. `/organizations`) it would have thrown an error because it would try to render `/organizations/img/diaper-base-logo-full-white-text.svg` rather than the correct path `/img/diaper-base-logo-full-white-text.svg`